docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+<style lang="scss" scoped>
+@media (min-width: 1536px) {
+  .custom-block.api-call > .response {
+    flex: 0 0 32%;
+  }
+}
+</style>
+
+# Managing relations through the REST API <BetaBadge />
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through [REST](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API by passing parameters in the body of the request:
+
+|  Parameter name | Description | Type of update |
+|-----------------|------------------|----------------|
+| [`connect`](#connect)       | Adds relation(s).<br /><br />Can be used in combination with `disconnect`.<br /><br />Can be used with [positional arguments](#relations-reordering) to define an order for relations.    | Partial
+| [`disconnect`](#disconnect)    | Removes relation(s).<br /><br />Can be used in combination with `connect`. | Partial
+| [`set`](#set)           | Sets relations, replacing all the existing ones.<br /><br />Must be used alone.  | Full
+
+Using `connect` allows adding [positional arguments](#relations-reordering) to define an order for relations.
+
+<br/>
+
+## `connect`
+
+Using `connect` in the body of a request performs a partial update, adding specified relations.
+
+`connect` accepts either a shorthand or a longhand syntax.
+
+| Syntax type | Syntax example |
+| ------------|----------------|
+| shorthand   | `connect: [2, 4]`
```

Comment:
How could we explain that we are referring to entity ids here?

Reply:
Good point, I'll improve this :)